app/src/main/AndroidManifest.xml

```diff
@@ -52,6 +54,12 @@
         </activity>
 
         <activity
+            android:name=".SignInActivity"
+            android:theme="@style/AppTheme"
+            />
```

Comment:
In V1 we won't support horizontal layouts. I would recommend to add these two lines here:
```
android:configChanges="orientation|keyboardHidden"
android:screenOrientation="portrait"
```